test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -292,6 +308,8 @@ BOOST_AUTO_TEST_CASE(events)
 	{
 		"name": "f",
 		"constant": false,
+		"view": false,
```

Comment:
If we want a single field, we should come up with a better name. I think  `constant` is definitely a no go and `promise` might be ok.

We have to consider the three states we have: `regular` (might find a better name), `view`, `pure`. They differ in their state changing ability.

`mutability`, `mutate`, `mutate`?